src/odemis/dataio/hdf5.py

```diff
@@ -541,11 +533,11 @@ def _read_image_info(group):
     # the pixel size (bandwidth) of the single px detector. The metadata includes MD_OUT_WL
     # (bandwidth of the monochromator), which is a tuple of (wl offset, wl offset + bandwidth).
 
-    # Spectrograph: The metadata includes MD_WL_POLYNOMIAL (conversion from px to wl), where
+    # Spectrograph: The metadata includes metadata in polynomial form (conversion from px to wl), where
     # the first entry corresponds to the wl offset "a" and the second to the coefficient "b" for the
     # polynomial of first order (wl = a + bx).
-    # Based on polynomial the wavelength values for all px can be calculated elsewhere
-    # (see odemis.util.spectrum.get_wavelength_per_pixel).
+    # Based on polynomial the wavelength values for all px can be calculated, converted to a list
+    # and saved in MD_WL_LIST.
```

Comment:
I'd suggest to make the text clearer that polynomial form is rare. Normally it's stored in list form. Something like:
```
Spectrograph: The metadata is normally stored in list form (of the same length as the C dimension). For old files, it sometimes had metadata in polynomial form, where the first entry corresponds to the wl offset "a" and the second to the coefficient "b" for the polynomial of first order (wl = a + bx). In such case, based on the polynomial, the wavelength values for all px are calculated, and saved in MD_WL_LIST.
```